src/component/button/HsiButton/HsiButton.tsx

```diff
@@ -155,21 +158,40 @@ export class HsiButton extends React.Component<HsiButtonProps, HsiButtonStatePro
     } else {
       pixel = map.getEventPixel(olEvt.originalEvent);
     }
+    let internalVectorFeatures: any = {};
```

Comment:
`any` should be replaced with the correct typing.

Reply:
Right, changed to `let internalVectorFeatures: {[name: string]: OlFeature[]} = {};` , will create a follow up pr